docs/templates/config.rst.j2

```diff
@@ -157,10 +157,11 @@ you can use the command line utility mentioned above (`ansible-config`) to brows
        :Version Added: {{ini_map['version_added']}}
 {%         endif %}
 {%         if ini_map['deprecated'] %}
-       :Deprecated in: {{ini_map['deprecated']['version']}}
-       :Deprecated detail: {{ini_map['deprecated']['why']}}
+       :Note: This config setting has been deprecated!
+       :Scheduled removal: {{ini_map['deprecated']['version']}}
+       :Deprecation reason: {{ini_map['deprecated']['why']}}
 {%           if ini_map['deprecated']['alternatives'] %}
-       :Deprecated alternatives: {{ini_map['deprecated']['alternatives']}}
+       :Non-deprecated alternatives: {{ini_map['deprecated']['alternatives']}}
```

Comment:
```suggestion
       :Current alternatives: {{ini_map['deprecated']['alternatives']}}
```

Reply:
When presenting options, contrast is important. I'd choose a different word because "Non-deprecated" is too close to "deprecated".